embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs

```diff
@@ -145,7 +145,7 @@ private void RegisterEventHandlers()
                 }
 
                 FlutterDesktopViewOnKeyEvent(
-                    View, e.Key.KeyPressedName, e.Key.KeyPressed, (uint)e.Key.KeyModifier, (uint)e.Key.KeyCode,
+                    View, e.Key.KeyPressedName, e.Key.KeyPressed, (uint)e.Key.KeyModifier, (uint)e.Key.KeyCode, e.Key.Time,
```

Comment:
I think you may also need below properties, these are may affect input panel behaviors on TV.
Could you please check?
```
DeviceSubClass
DeviceClass
DeviceName
```

Reply:
Thanky you for check.
I add more key information.